packages/types/types/index.d.ts

```diff
@@ -48,6 +49,13 @@ type RefPropType = (() => any) | ReactRefObject
  */
 type AbsoluteProps = TopRightBottomLeft
 
+export interface BackgroundImageProps extends WidthProps {
+  height?: string
+  /** URL of background image */
```

Comment:
Is this comment necessary?

Reply:
I figured it couldn't hurt to be explicit because it's not a standard name like `src`